jupytercalpoly/Jupyter-Bifrost
# Axis Scaling

**Added axis scaling for quantitative values in the `FilterScreen`**

## Changes
- Dropdown select field in filter screen

## Testing
- Go to the filter screen for a quantitative variable.
- Apply the axis scale
- Verify that the axis has been altered in scale
